vignettes/Guidelines_Contribution_Dataset-formatting.Rmd

```diff
@@ -0,0 +1,235 @@
+- `cell_x` and `cell_y`: position of the cell centroid on the image. These columns are used as `SpatialCoords` when converting to a [SpatialExperiment](https://bioconductor.org/packages/SpatialExperiment) object.
+
+### rowData
+
+The `rowData` entry of the `SingleCellExperiment` is a `DataFrame` that contains marker (protein, RNA, probe) information.
+
+The following columns are required in the `rowData` entry:
+
+- `channel`: a unique integer that maps to the channels of the associated multichannel images.
+- `metal`: the metal isotope used for detection, formatted as { `ChemicalSymbol``IsotopeMass`} (e.g., `Y89`, `In115`, `Yb176`, `Bi209`).
+- `name`: full name of the marker.
+- `short_name`: abbreviated marker name, preferably following the official [UniProt](https://www.uniprot.org) nomenclature.
+
+For the `name` and `short_name` columns, the following guidelines apply:
+
+* In `short_name`, all dashes, dots and spaces should removed or replaced with underscores.
+* For post-translationally modified proteins:
+  * Prefix the `name` with the modification type (e.g., `phospho-`, `methyl-`) and suffix it with the modified aminoacids (e.g., `[S28]`).
+  * Prefix the `short_name` with an abbreviation of the modification type (e.g., `p_`, `me_`) and do not indicate the modified aminoacids, unless there is a possible confusion with another target in the dataset.
+
+```{r echo=FALSE, results='hide'}
+marker_names <- data.frame(
+  name = c(
+    "Carbonic anhydrase IX",
+    "CD3 epsilon",
+    "CD8 alpha",
+    "E-Cadherin",
+    "cleaved-Caspase3 + cleaved-PARP",
+    "Cytokeratin 5",
+    "Forkhead box P3",
+    "Glucose transporter 1",
+    "Histone H3",
+    "phospho-Histone H3 [S28]",
+    "Ki-67",
+    "Myeloperoxidase",
+    "Programmed cell death protein 1",
+    "Programmed death-ligand 1",
+    "phospho-Rb [S807/S811]",
+    "Smooth muscle actin",
+    "Vimentin",
+    "Ir191",
+    "Ir193"
+  ),
+  short_name = c(
+    "CA9",
+    "CD3e",
+    "CD8a",
+    "CDH1",
+    "cCASP3_cPARP",
+    "KRT5",
+    "FOXP3",
+    "SLC2A1",
+    "H3",
+    "p_H3",
+    "Ki67",
+    "MPO",
+    "PD_1",
+    "PD_L1",
+    "p_Rb",
+    "SMA",
+    "VIM",
+    "DNA1",
+    "DNA2"
+  )
+)
+```
+
+```{r echo=FALSE, results='asis'}
+knitr::kable(
+  marker_names,
+  caption = "'name' and 'short_name' examples for some commonly used markers"
+)
+```
+
+Finally, the `short_name` column should be used to name the rows of the `rowData` entry and as `rownames(sce)`.
+
+### assays
+
+The `assays` slot of the `SingleCellExperiment` contains counts matrices representing marker expression levels per cell and channel.
+
+It should at least contain a `counts` matrix with raw ion counts. The `assays` slot can also contain additional matrices with commonly used counts transformations, or counts transformations that were used in the article describing the dataset. For instance:  
+
+* `exprs`: asinh-transformed counts, with a cofactor of 1.
```

Comment:
Are different co-factors or transformations allowed?

Reply:
Changed the wording to say a cofactor of 1 is typically used for IMC data. Also added that the type of transformation should be described in the `man` pages of the `.R` function.